test/conftest.py

```diff
@@ -39,14 +40,15 @@ def pytest_addoption(parser: pytest.Parser) -> None:
     parser.addoption('--run_id', action='store', default=None, help='Run id for the test run')
     parser.addoption('--byte-limit', action="store", default=randint(0, 2000), type=int,
                      help="Specific byte limit for failure injection (random by default)")
+    parser.addoption("--gather-metrics", action=BooleanOptionalAction, default=False, help='Switch on gathering cgroup metrics')
 
     # Following option is to use with bare pytest command.
     #
     # For compatibility with reasons need to run bare pytest with  --test-py-init option
     # to run a test.py-compatible pytest session.
     #
     # TODO: remove this when we'll completely switch to bare pytest runner.
-    parser.addoption('--test-py-init', action='store_true', default=False,
+    parser.addoption('--test-py-init', action='store_false',
```

Comment:
What does it have to do with "adding the possibility to gather resource metrics for C++ tests"?